packages/lodestar/src/eth1/eth1DepositDataTracker.ts

```diff
@@ -99,6 +105,11 @@ export class Eth1DepositDataTracker {
     state: CachedBeaconStateAllForks,
     eth1DataVote: phase0.Eth1Data
   ): Promise<phase0.Deposit[]> {
+    // No new deposits have to be included, continue
+    if (eth1DataVote.depositCount === state.eth1DepositIndex) {
+      return [];
+    }
```

Comment:
@dapplion do we also want to try/catch the below code to guarantee we can produce block even if there are errors in `getDeposits()`?

Reply:
No, because if don't include all the deposits necessary the block is invalid